data/tutorial_tests/asterisk_ari/asterisk/etc/asterisk/extensions.conf

```diff
@@ -2,7 +2,7 @@
  exten => _1XXX,1,NoOp()
  same => n,Set(CGRMaxSessionTime=0); use it to disconnect automatically the call if CGRateS is not active
  same => n,DumpChan()
- same => n,Stasis(cgrates_auth,cgr_reqtype=*prepaid,cgr_supplier=supplier1,cgr_subsystems=*accounts*attributes*resources*stats*suppliers*thresholds)
+ same => n,Stasis(cgrates_auth,cgr_reqtype=*prepaid,cgr_supplier=supplier1,"cgr_flags=*accounts,*attributes,*resources,*stats,*suppliers,*thresholds")
```

Comment:
Do we need to specify flags between ""?

Reply:
Yes because for some reason asterisk will split  the string after `,` and we will not obtain all the flags.